### DISCUSSION TITLE
Animestyle Clover theme.

### DISCUSSION THREAD

**Discussion by valeriumussr** [chris1111/My-Github-Pages#2] [Show and tell]
Hi Chris! I made an anime theme for clover and I really want you to test it and add it to the main repository of themes for clover.
I hope you like it.

[animestyle.zip] (https://github.com/chris1111/My-Github-Pages/files/10478644/animestyle.zip)

![screenshot] (https://user-images.githubusercontent.com/108763753/214013861-2423911e-167f-4a7e-ab75-1471a1d964ab.png)

**Comment by chris1111** [comment]
> Hi Chris! I made an anime theme for clover and I really want you to test it and add it to the main repository of themes for clover. I hope you like it.
> 
> [animestyle.zip] (https://github.com/chris1111/My-Github-Pages/files/10478644/animestyle.zip)
> 
> ![screenshot] (https://user-images.githubusercontent.com/108763753/214013861-2423911e-167f-4a7e-ab75-1471a1d964ab.png)



Ive test your themes Its beautifull Congrat great jog! 👍
Your only have to[ PullRequest ](https://github.com/valeriumussr/CloverThemes/pulls)because you have fork the repos 

Good jod anyway
![screenshot0](https://user-images.githubusercontent.com/6248794/214018305-ec7a4a31-d7c6-4d33-aed1-6291c8005ca4.png)

**Reply by valeriumussr** [reply]
How i make it?

**Reply by valeriumussr** [reply]
I did it

**Reply by chris1111** [reply]
> How i make it?


[Done accepted congrats ](https://github.com/CloverHackyColor/CloverThemes/pull/7#event-8333018853)

**Reply by valeriumussr** [reply]
Thank you very much! if I come up with something new, I will definitely send it to you
